### ISSUE TITLE
Navigation goes to wrong place instead of showing errors if network connectivity lost

### ISSUE THREAD

**Issue by jimrandomh** [LessWrong2/Lesswrong2#975]
Go to a sequence. Read a few posts. Disconnect your network connection, then try to navigate further. You can navigate to the sequence index and to posts you've previously loaded in that tab, but if you click on a link to a post you haven't loaded before, it... sends you to the wrong post.

**Comment by Discordius** [comment]
Yeah, the wrong post thing is weird. I noticed that in a few different places.
